### ISSUE TITLE
On further rationalizing the ROI metrics in the OCEANDAO

### ISSUE THREAD

**Issue by TimDaub** [oceanprotocol/oceandao#13]
:mega:  Since there was a discussion about ROI yesterday, I've wrote down the main issues I've had with ROI calculations over the course of my membership in the oceanDAO: https://hackmd.io/OFp3-Ph_RcyJGlQepi15zQ?view

**Comment by trentmc** [comment]
Hi Tim! Thanks for raising this issue. 

You have been heard.

OceanDAO has moved away from "ROI > 1" and towards "value add" [details here](https://blog.oceanprotocol.com/introducing-oceandao-working-groups-db920b14a5d8).

Therefore - would you consider this issue able to close?

**Comment by TimDaub** [comment]
Great read. I also looked into the updated project criteria: https://github.com/oceanprotocol/oceandao/wiki/project-criteria

In some ways, I'm happy that we stopped further rationalizing the ROI metric and instead softened it. It's true what is said in the blog post: It may be super hard to objectify certain actions as "ROI > 1" (or a dollar value), e.g. maintaining the docs.

I'm already curious to see how the new criteria will change projects' developments.
